src/data/screenshots.json

```diff
@@ -5,13 +5,18 @@
     },
     "section-main": {
         "headline": {
-            "title": "Take a Look (Version 2.19)"
+            "title": "Take a Look (Version 2.20)"
         },
         "content": {
             "textblock": [
-                "Browse through the screenshots to get an overview of the features offered by the Corona-Warn-App with the release of Version 2.19 (available since March 16th, 2022). Please note that the screenshots may vary slightly between Android and iOS due to technical reasons.",
+                "Browse through the screenshots to get an overview of the features offered by the Corona-Warn-App with the release of Version 2.20 (available since ????? ??th, 2022). Please note that the screenshots may vary slightly between Android and iOS due to technical reasons.",
                 "With this update we provide new and enhanced features:",
-                "<b>Certificate Assignment:</b> The assignment of certificates to people has been made more error-tolerant. Minor differences in the spelling of first and last names are ignored, so affected certificates can still be assigned to the same person.",
+                "<b>Shortened display of increased risk:</b> Increased risk after a risk encounter is only displayed for 10 days instead of 14.",
+                "<b>Risk display after positive test result:</b> Even if you have registered a positive test in the Corona-Warn-App, you will now see the red tile for increased risk after a risk encounter. Seven days after the positive test, your risk status will be permanently displayed again.",
+                "<b>Notification about new risk encounter:</b> If you had another encounter with increased risk after a risk encounter, you will now see a notification about it.",
+                "<b>Notification about status change:</b> If your status has changed (e.g. by adding a certificate, or because the rules have changed) you will be informed about it on the certificate tab.",
+                "<b>Certificates of recovery without validity dates:</b> Validity dates are no longer displayed for certificates of recovery, since validity is determined by local rules. Instead, the date of the first positive test is now displayed.",
+                "<b>Duplicate QR code removed:</b> From the details screen for your person, the QR code for your currently used certificate has been removed. The QR code is also located on the certificate overview, so it is not needed on the details screen.",
```

Comment:
"From the details screen for your person, the QR code for your currently used certificate has been removed." =>

The QR code for your currently used certificate has been removed from the details screen for your person."

The original sentence is not proper English!

Reply:
Fixed in https://github.com/corona-warn-app/cwa-website/pull/2543/commits/33ddb2e34ce36eb63feb55bd78cb292ca6c85272, thanks!